keyboards/qvex/lynepad/config.h

```diff
@@ -1,11 +1,10 @@
 /*
-Copyright 2020 KemoNine
+Copyright 2023 QVEX Tech
```

Comment:
```suggestion
Copyright 2020 KemoNine
Copyright 2023 QVEX Tech
```

Reply:
I am completely fine with crediting the original implementation of the Lynepad. However, this seems like a strange way to do that. This lynepad2 was a complete rework and it is not based on the original implementation. Apart from retaining the original Kemonine's keymap and modifying it to work with the new rework.